libs/domain-service/src/registries/index.ts

```diff
@@ -0,0 +1,57 @@
+import { Registry } from "../types";
+
+const REGISTRIES: Registry[] = [
```

Comment:
Should we import this one later in an other file ?

Reply:
Yup, the goal is to have this thing from a backend instead once they will release it. 
That's why the `getRegistries` method is async, so we don't have to add async/awaits everywhere afterwards once it's coming from that backend. 👌